cmd/lncli/commands.go

```diff
@@ -2540,6 +2573,25 @@ func parseChanBackups(ctx *cli.Context) (*lnrpc.RestoreChanBackupRequest, error)
 			},
 		}, nil
 
+	case ctx.IsSet("single_file"):
+		packedSingle, err := ioutil.ReadFile(ctx.String("single_file"))
+		if err != nil {
+			return nil, fmt.Errorf("unable to decode multi packed "+
+				"backup: %v", err)
+		}
+
+		return &lnrpc.RestoreChanBackupRequest{
+			Backup: &lnrpc.RestoreChanBackupRequest_ChanBackups{
+				ChanBackups: &lnrpc.ChannelBackups{
+					ChanBackups: []*lnrpc.ChannelBackup{
```

Comment:
This will trigger the "line too long" linter.
Can be shortened by:
```
					ChanBackups: []*lnrpc.ChannelBackup{{
						ChanBackup: packedSingle,
					}},
```

Reply:
Resolved.